Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -72,19 +87,63 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
             var indexList = new List<int>();
             List<int> spaceIndices = new List<int>();
 
+            bool spaceMet = false;
+
             for (var compareStringIndex = 0; compareStringIndex < fullStringToCompareWithoutCase.Length; compareStringIndex++)
             {
+                if (currentAcronymQueryIndex >= queryWithoutCase.Length
+                    || allQuerySubstringsMatched && acronymScore < (int) UserSettingSearchPrecision)
+                    break;
 
                 // To maintain a list of indices which correspond to spaces in the string to compare
                 // To populate the list only for the first query substring
-                if (fullStringToCompareWithoutCase[compareStringIndex].Equals(' ') && currentQuerySubstringIndex == 0)
-                {
+                if (fullStringToCompareWithoutCase[compareStringIndex] == ' ' && currentQuerySubstringIndex == 0)
                     spaceIndices.Add(compareStringIndex);
+
+                // Acronym check
+                if (char.IsUpper(stringToCompare[compareStringIndex]) ||
+                    char.IsNumber(stringToCompare[compareStringIndex]) ||
+                    char.IsWhiteSpace(stringToCompare[compareStringIndex]) ||
+                    spaceMet)
```

Comment:
@taooceros what's this 'spaceMet' variable trying to solve and is this needed for translation?

Reply:
I think it is used to mark character after space to be acronym even when it is lower case.

Reply:
Ok cool, I thought so as well. I am thinking of using substring index = 0 to replace this, since the start of substring char is always acronym

Reply:
Have we split the string to compare? I think we only split the query string.

Reply:
Oh we didn't, lol ok nvm then

Reply:
Alternatively then, we can go back an index to check if it is space

Reply:
Yeah, that's feasible.